charts/quickwit/templates/job-create-indices.yaml

```diff
@@ -35,13 +35,20 @@ spec:
         imagePullPolicy: {{ $.Values.image.pullPolicy }}
         command: ["/bin/bash","-c","quickwit index describe --index {{ .index_id }} --endpoint ${QW_CLUSTER_ENDPOINT} || quickwit index create --index-config {{ .index_id }}.yaml --endpoint ${QW_CLUSTER_ENDPOINT}"]
         env:
-          - name: QW_CLUSTER_ENDPOINT
-            value: http://{{ include "quickwit.fullname" $ }}-metastore.{{ $.Release.Namespace }}.svc.cluster.local:7280
-          {{- include "quickwit.metastore.environment" $ | nindent 10 }}
+          {{- include "quickwit.environment" $ | nindent 10 }}
           {{- range $key, $value := $.Values.bootstrap.extraEnv }}
           - name: "{{ $key }}"
             value: "{{ $value }}"
           {{- end }}
+          {{- if or (.Values.environmentFrom) (.Values.bootstrap.extraEnvFrom) }}
```

Comment:
Because it's all in a `range`, it should be `{{- if or ($.Values.environmentFrom) ($.Values.bootstrap.extraEnvFrom) }}` to be in the correct context.